src/layout.jl

```diff
@@ -284,3 +284,288 @@ function _spectral(A::SparseMatrixCSC)
     index = sortperm(real(eigenvalues))[2:3]
     return real(eigenvectors[:, index[1]]), real(eigenvectors[:, index[2]])
 end
+
+# This layout algorithm is copy from [IainNZ](https://github.com/IainNZ)'s [GraphLayout.jl](https://github.com/IainNZ/GraphLayout.jl)
```

Comment:
Is there any advantage in moving this function? Maybe it would be easier to read if we keep it in a separate file.

Reply:
I figured that since we had a `layout.jl` file with several layouts, I'd just add it there. However, we could have a layout folder and have a separate file for each layout. What are your thoughts?